Tests/Integration/MSFT_xWebBindingInformation.Tests.ps1

```diff
@@ -1,59 +1,83 @@
-﻿$DSCResourceName = 'MSFT_xWebsite'
+﻿# should check for the server OS
+if($env:APPVEYOR_BUILD_VERSION)
+{
+     Add-WindowsFeature Web-Server -Verbose
+}
 
-$Splat = @{
-    Path = $PSScriptRoot
-    ChildPath = "..\..\DSCResources\$DSCResourceName\$DSCResourceName.psm1"
-    Resolve = $true
-    ErrorAction = 'Stop'
+$DSCModuleName   = 'xWebAdministration'
+
+$moduleRoot = "${env:ProgramFiles}\WindowsPowerShell\Modules\$DSCModuleName"
+
+if(-not (Test-Path -Path $moduleRoot))
+{
+    $null = New-Item -Path $moduleRoot -ItemType Directory
+}
+else
+{
+    # Copy the existing folder out to the temp directory to hold until the end of the run
+    # Delete the folder to remove the old files.
+    $tempLocation = Join-Path -Path $env:Temp -ChildPath $DSCModuleName
+    Copy-Item -Path $moduleRoot -Destination $tempLocation -Recurse -Force
+    Remove-Item -Path $moduleRoot -Recurse -Force
+    $null = New-Item -Path $moduleRoot -ItemType Directory
 }
-$DSCResourceModuleFile = Get-Item -Path (Join-Path @Splat)
 
-# should check for the server OS
-if($env:APPVEYOR_BUILD_VERSION)
+Copy-Item -Path $PSScriptRoot\..\..\* -Destination $moduleRoot -Recurse -Force -Exclude '.git'
+
+if (Get-Module -Name $DSCModuleName -All)
 {
-  Add-WindowsFeature Web-Server -Verbose
+    Get-Module -Name $DSCModuleName -All | Remove-Module
 }
 
-Import-Module -Name $DSCResourceModuleFile.FullName -Force
+Import-Module -Name $(Get-Item -Path (Join-Path $moduleRoot -ChildPath 'xWebadministration.psd1')) -Force
 
-Describe 'MSFT_xWebBindingInformation' {
-    It 'Should be able to get xWebsite' -test {
-        # just a good idea.  
-        # I thought it might force the classes to register, but it does not.
-        $resources = Get-DscResource -Name xWebsite
-        $resources.count | should be 1
-    }
+# This fixes a bug in AppVeyor where it was picking up duplicates of the code
+if (($env:PSModulePath).Split(';') -ccontains $pwd.Path)
+{
+    $script:tempPath = $env:PSModulePath
+    $env:PSModulePath = ($env:PSModulePath -split ';' | Where-Object {$_ -ne $pwd.path}) -join ';'
+}
+
```

Comment:
And here :) :
```powershell
# Preserve and set the execution policy so that the DSC MOF can be created
$executionPolicy = Get-ExecutionPolicy
if ($executionPolicy -ne 'Unrestricted')
{
    Set-ExecutionPolicy -ExecutionPolicy Unrestricted -Force
    $rollbackExecution = $true
}
```